### DISCUSSION TITLE
FluentAutoComplete not working

### DISCUSSION THREAD

**Discussion by imroatulfaizah** [microsoft/fluentui-blazor#2879] [Q&A]
so, i have a form like this

![image](https://github.com/user-attachments/assets/792e449e-d246-4606-a090-b63b86644fd8)

when I click the X function (clear), it's freeze.
what should i do?

I browsing and get this information:
![image](https://github.com/user-attachments/assets/09f67108-bdb0-4bb7-a801-dc443a9d682f)

we use 

`<div class="col-md-3">
    <FluentAutocomplete TOption="ms_data"
                        AutoComplete="off"
                        Label="Status Flag"
                        Placeholder="Select Status Flag"
                        OnOptionsSearch="@OnSearchAsyncStatusFlag"
                        OptionText="@(item => item.description)"
                        OptionValue="@(item => item.code)"
                        MaximumSelectedOptions="1"
                        Style="width:100%;"
                        Id="ListStatusFlag"
                        @bind-SelectedOptions="@ListStatusFlag"
                        @bind-SelectedOptions:after="OnStatusFlagSelected"
                        @bind-Value="@Content.status_flag"
                            Required />
</div>`
`<EditForm EditContext="editContext">
    --code
</EditForm>`

**Comment by imroatulfaizah** [comment]
<EditForm EditContext="editContext">

i use editcontext

**Reply by vnbaaij** [reply]
Are you using SSR?

It helps if you post code examples that people can copy and run. Just showing the code for that one tag is not really going to help to assess where the problem is.

**Reply by Lindsay-Mathieson** [reply]
> Are you using SSR?
> 
> It helps if you post code examples that people can copy and run. Just showing the code for that one tag is not really going to help to assess where the problem is.

I just ran into this with my WPF Hybrid app, dotnet 8 - a 100 item dropdown with max 1 select, vs 4.10.3, If I select, then click the far right x to clear the box, it doesn't clear and the drop down stops working.

Reduced the max items down to 9 and it seemed ok after that. Going to try rolling back to 4.10.2 to see if that fixes it.

Managed to reproduce it with the demo site many items sample:

* https://www.fluentui-blazor.net/#manyitems

Selected multiple items, then tried to clear it and it froze up. Edge on windows 11.
